crates/router/src/connector/adyen.rs

```diff
@@ -367,7 +367,7 @@ impl
     ) -> CustomResult<RequestContent, errors::ConnectorError> {
         let authorize_req = types::PaymentsAuthorizeRouterData::foreign_from((
             req,
-            types::PaymentsAuthorizeData::from(req),
+            types::PaymentsAuthorizeData::foreign_from(req),
```

Comment:
These can all be `From` implementations itself, since all the types now reside in the `hyperswitch_domain_models` crate, right?

Reply:
No RouterData alias type definition `SetupMandateRouterData` exists in router_crate. Once that is moved to `hyperswitch_domain_models`, We can move the From trait implementation to `hyperswitch_domain_models`

Reply:
Yeah, can't we move the type aliases as well to the `hyperswitch_domain_models` crate?

Reply:
We can. I would like to take that up in a new PR. Along with moving all From and TryFrom implementations to `hyperswitch_domain_models`.